_benchmark/user-guide/concepts.md

```diff
@@ -25,6 +25,93 @@ A workload is a specification of one or more benchmarking scenarios. A workload
 - One or more data streams that are ingested into indexes.
 - A set of queries and operations that are invoked as part of the benchmark.
 
+## Throughput and latency
+
+At the end of each test, OSB produces a table that summarizes the following 
+
+- [Service time](#service-time) 
+- Throughput
+- [Latency](#latency)
+- The error rate for each task or OpenSearch operation that ran 
+
+While the definition for _throughput_ remains consistent with other client-server systems, the definitions for `service time` and `latency` differ from most client-server systems in the context of OSB. The following table compares how service time and latency work with OSB versus the common definition of each term for a client-server system.
+
+| Metric | Common definition | **OSB Definition**	|
+| :--- | :--- |:--- |
+| **Throughput** | The number of operations completed in a given period of time	| The number of operations completed in a given period of time	|
+| **Service time**	|The time the server takes to process a request, from the point it receives the request to the point the response is returned. </br></br> It includes the time spent waiting in server-side queues, but _excludes_ network latency, load-balancer overhead, and deserialization/serialization. | The time it takes for the `opensearch-py` to send a request to and receive a response from the OpenSearch cluster. </br> </br> It includes the time it takes for the server to process a request, and also _includes_ network latency, load-balancer overhead, and deserialization/serialization.  |
+| **Latency** | The total time, including the service time and the time the request waited before responding. | The total the request waited before responding based on the `target-throughput` set and any other delays that occur before the request responds. |
```

Comment:
`The total the request waited before responding based on the "target-throughput" set and any other delays that occur before the request responds.` -> `The total time the request waited before receiving the response in addition to any other delays that occur before the request is sent. Is based on target-throughput that is set by the user.`